adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -1098,10 +1120,33 @@ protected void onPostExecute(TokenTaskResult result) {
                             userinfo.getDisplayableId());
                 }
 
+                if (null != result.mTaskResult.getCliTelemInfo()) {
```

Comment:
Just to confirm, these changes in this class are ADAL only related changes right? I mean broker has nothing to do with it ?

Reply:
That's correct - that's for double checking.